lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
For "Unavailable on mobile" and "only on web" to mean the same thing, it would have to be True and False (or False and True) respectively. I think it would be less confusing if they were worded such that they are either both true or both false.